meta-resin-sumo/recipes-kernel/linux/linux-firmware_git.bbappend

```diff
@@ -0,0 +1,17 @@
+inherit balena-linux-firmware
```

Comment:
I wouldn't prefix this filename with balena

Reply:
hmm. true. this isn't just an internal os one that we can avoid prefixing with balena. ok then.